packages/react-core/vite.config.ts

```diff
@@ -16,7 +16,12 @@ export default defineConfig(({ mode }) => {
       dts({
         outDir: "dist/types",
       }),
-      react(),
+      react({
+        jsxRuntime: "classic",
+        babel: {
+          plugins: ["react-require"],
```

Comment:
Is this just for convenience or necessary because of something else in the build? Just curious. I like the idea of having it either way because it'd be easy to forget we have to explicitly import React.

Reply:
Convenience because since React 17 everyone got used to never writing `import React` 🙃